### PR TITLE
feat(syncInfo): build syncinfo orders objects

### LINKED ISSUE #35: Set syncinfo after exporting order via CSV

### PR DESCRIPTION
**Pull request by hisabimbola**
* creates a JSON file that contains a collections of orders that needs to be synced
* this feature only works when using SFTP
* the file is uploaded to the SFTP server

Fixes #35

### CODE REVIEW

File: src/coffee/run.coffee [hunk 1]
```diff
@@ -226,13 +255,21 @@ ensureCredentials(argv)
...
               sftpClient.safePutFile(sftp, "#{@outputDir}/#{filename}", "#{sftpTarget}/#{filename}")
               .then =>
-                xml = _.find @orderReferences, (r) -> r.name is filename
-                if xml
-                  logger.debug "About to sync order #{filename}"
-                  orderExport.syncOrder xml.entry, filename
+                if exportType.toLowerCase() is 'csv' and argv.createSyncActions
+                  createSyncOrders(@orderReferences[0].fileName)
+                    .then (orders) =>
+                      orderJsonFile = "tobeSyncOrders.json" # Can be renamed??
```
Comment (Siilwyn) [inline src/coffee/run.coffee]: Can be renamed?

Comment (hisabimbola) [inline src/coffee/run.coffee]: lol, yeah...wanted to discuss with Nima about somethings

File: src/coffee/run.coffee [hunk 2]
```diff
@@ -258,10 +257,18 @@ ensureCredentials(argv)
                 if exportType.toLowerCase() is 'csv' and argv.createSyncActions
                   createSyncOrders(@orderReferences[0].fileName)
                     .then (orders) =>
-                      orderJsonFile = "tobeSyncOrders.json" # Can be renamed??
-                      fs.writeFileAsync("#{@outputDir}/#{orderJsonFile}", JSON.stringify(orders,null,2)).then =>
-                        logger.debug "Uploading #{@outputDir}/#{orderJsonFile}"
-                        sftpClient.safePutFile(sftp, "#{@outputDir}/#{orderJsonFile}", "#{sftpTarget}/#{orderJsonFile}")
+                      if orders.length
+                        if argv.fileWithTimestamp
```
Comment (Siilwyn) [inline src/coffee/run.coffee]: Think this argument can be removed to simplify, I wouldn't see a use-case for having the file without a timestamp. Remember, settings are harmful!

Comment (hisabimbola) [inline src/coffee/run.coffee]: I was following the already set standards for the tool, but yeah settings are harmful 👿👺

Comment (Siilwyn) [inline src/coffee/run.coffee]: Ah well let's merge then ╰( ⁰ ਊ ⁰ )━☆ﾟ.*･｡ﾟ

Removing it would be a major bump we can do with a separate PR.
